### DISCUSSION TITLE
A November to Remember - Actions and Packages Check-in 🛰️

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#38274] [Actions]
Hello 👋  everyone! The month of November 📆  is here and we are so excited for GitHub Universe 🛰️ which is just a few days away and it's going to be more robust than ever with  2️⃣  great ways to experience everything the global developer event has to offer: an in-person experience in San Francisco, and a dynamic [virtual experience](https://register.githubuniverse.com/). 

![Screenshot 2022-11-01 at 3 56 30 PM](https://user-images.githubusercontent.com/95834130/199972588-6a748f72-0c2d-4d6f-a691-c6954008be6c.png)

We’ve got over 80+ sessions and 120+ speakers covering what’s new, exciting, and beyond . 🤯  Save your favorite GitHub Universe sessions on your calendar 📅  and [build your own schedule](https://watch.githubuniverse.com/schedule) to get the most out of your conference experience. For more details, check out this [blogpost](https://github.blog/2022-10-27-choose-your-own-github-universe-2022-adventure/) by [@lclindeman](https://github.com/lclindeman) and I will be looking forward to learning with you all on November 9, 10, and beyond! 🚀 

### [Sigstore is now generally available](https://github.blog/2022-10-25-why-were-excited-about-the-sigstore-general-availability/) - Get secure software supply chain security with GitHub Actions 🔐 

Using GitHub Actions with Sigstore has now become easier. If you don’t know much about Sigstore, it can generate a signing certificate for your build without having to maintain a private key. Sigstore calls this [keyless signing](https://github.com/sigstore/cosign/blob/main/KEYLESS.md), and it’s  fantastic for  lowering the barrier to entry so more projects can sign their builds. Sigstore general availability is an important milestone to achieving such goals. 🥅 

### [Cache Management UX](https://github.blog/changelog/2022-10-20-manage-caches-in-your-actions-workflows-from-web-interface/) - GitHub Actions are now even faster  ⚡ 

We have newly designed a web interface for managing Actions cache that enables you to speed up your GitHub Actions workflows and make them better with more transparency and control over their cache usage within the GitHub repositories.

![Screenshot 2022-11-02 at 3 14 27 PM](https://user-images.githubusercontent.com/95834130/199495191-b9e95110-3d72-40aa-aa3c-145cdbc25913.png)

In addition to the Cache Management UX that we have now enabled, you could also use our [Cache APIs](https://docs.github.com/en/rest/actions/cache) or install the [GitHub CLI extension for Actions cache](https://github.com/actions/gh-actions-cache#readme) to manage your caches from your terminal. 📝 

Learn more about [dependency caching to speed up your Action workflows](https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows).

### [Revamped navigation for Actions](https://github.blog/2022-10-20-improving-navigation-for-github-actions/) 💞 

Another UX masterpiece to help you use Actions effortlessly. We’ve introduced a 🆕 navigation formanagingyour Actions experience, improving discoverability and accessibility, as well as opening up future feature opportunities. No wonder this is my new favorite 🤟  Actions new feature. There are a couple of great additions that you don’t want to miss out on, read the [blogpost](https://github.blog/2022-10-20-improving-navigation-for-github-actions/) to learn more! 🔖 

### [Dependabot now updates comments in GitHub Actions workflows referencing action versions](https://github.blog/changelog/2022-10-31-dependabot-now-updates-comments-in-github-actions-workflows-referencing-action-versions/) 🤖 

With this [change](https://github.com/dependabot/dependabot-core/pull/5951), Dependabot will now update the semver version in comments when updating Actions workflows with a commit SHA (referencing) version.

## From the Community 🤝 

In the coming week, I’ll be moderating the community for all questions related to Universe and would love your help and expertise in answering these unanswered topics. 🙇 

- Do you know a way to [make each workflow run Unique without disabling the workflows](https://github.com/orgs/community/discussions/37488)? 

- [Accessing both the keys and values of an object in a matrix strategy](https://github.com/orgs/community/discussions/37566)

- [Running specific strategy on a different host](https://github.com/orgs/community/discussions/37815)

- [How to validate that the user that issued a comment belongs to a specific github team](https://github.com/orgs/community/discussions/37769) 

- [Referencing the job calling a reusable workflow](https://github.com/orgs/community/discussions/37414)

- [Resolving an issue where the environment shown in GitHub only shows the latest commit and not a list of commits](https://github.com/orgs/community/discussions/37391) 

- [How to Creare multi-variant matrix strategy from a step](https://github.com/orgs/community/discussions/37478)

- [Step output playground - how to properly escape and write access to output variables](https://github.com/orgs/community/discussions/37477) ☮️

**Comment by 3kh0** [comment]
Yippee! 🚀

**Comment by Wgdan1** [comment]
Wgdan203040@gmail.com

**Comment by JOHAN10p** [comment]
cooooool

**Comment by ghost** [comment]
> Hello 👋 everyone! The month of November 📆 is here and we are so excited for GitHub Universe 🛰️ which is just a few days away and it's going to be more robust than ever with 2️⃣ great ways to experience everything the global developer event has to offer: an in-person experience in San Francisco, and a dynamic [virtual experience](https://register.githubuniverse.com/).
> 
> ![Screenshot 2022-11-01 at 3 56 30 PM](https://user-images.githubusercontent.com/95834130/199972588-6a748f72-0c2d-4d6f-a691-c6954008be6c.png)
> 
> We’ve got over 80+ sessions and 120+ speakers covering what’s new, exciting, and beyond . 🤯 Save your favorite GitHub Universe sessions on your calendar 📅 and [build your own schedule](https://watch.githubuniverse.com/schedule) to get the most out of your conference experience. For more details, check out this [blogpost](https://github.blog/2022-10-27-choose-your-own-github-universe-2022-adventure/) by [@lclindeman](https://github.com/lclindeman) and I will be looking forward to learning with you all on November 9, 10, and beyond! 🚀
> 
> ### [Sigstore is now generally available](https://github.blog/2022-10-25-why-were-excited-about-the-sigstore-general-availability/) - Get secure software supply chain security with GitHub Actions 🔐
> Using GitHub Actions with Sigstore has now become easier. If you don’t know much about Sigstore, it can generate a signing certificate for your build without having to maintain a private key. Sigstore calls this [keyless signing](https://github.com/sigstore/cosign/blob/main/KEYLESS.md), and it’s fantastic for lowering the barrier to entry so more projects can sign their builds. Sigstore general availability is an important milestone to achieving such goals. 🥅
> 
> ## From the Release Radar 🚨
> ### [Cache Management UX](https://github.blog/changelog/2022-10-20-manage-caches-in-your-actions-workflows-from-web-interface/) - GitHub Actions are now even faster ⚡
> We have newly designed a web interface for managing Actions cache that enables you to speed up your GitHub Actions workflows and make them better with more transparency and control over their cache usage within the GitHub repositories.
> 
> ![Screenshot 2022-11-02 at 3 14 27 PM](https://user-images.githubusercontent.com/95834130/199495191-b9e95110-3d72-40aa-aa3c-145cdbc25913.png)
> 
> In addition to the Cache Management UX that we have now enabled, you could also use our [Cache APIs](https://docs.github.com/en/rest/actions/cache) or install the [GitHub CLI extension for Actions cache](https://github.com/actions/gh-actions-cache#readme) to manage your caches from your terminal. 📝
> 
> Learn more about [dependency caching to speed up your Action workflows](https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows).
> 
> ### [Revamped navigation for Actions](https://github.blog/2022-10-20-improving-navigation-for-github-actions/) 💞
> Another UX masterpiece to help you use Actions effortlessly. We’ve introduced a 🆕 navigation formanagingyour Actions experience, improving discoverability and accessibility, as well as opening up future feature opportunities. No wonder this is my new favorite 🤟 Actions new feature. There are a couple of great additions that you don’t want to miss out on, read the [blogpost](https://github.blog/2022-10-20-improving-navigation-for-github-actions/) to learn more! 🔖
> 
> ### [Dependabot now updates comments in GitHub Actions workflows referencing action versions](https://github.blog/changelog/2022-10-31-dependabot-now-updates-comments-in-github-actions-workflows-referencing-action-versions/) 🤖
> With this [change](https://github.com/dependabot/dependabot-core/pull/5951), Dependabot will now update the semver version in comments when updating Actions workflows with a commit SHA (referencing) version.
> 
> ## From the Community 🤝
> In the coming week, I’ll be moderating the community for all questions related to Universe and would love your help and expertise in answering these unanswered topics. 🙇
> 
> * Do you know a way to [make each workflow run Unique without disabling the workflows](https://github.com/orgs/community/discussions/37488)?
> * [Accessing both the keys and values of an object in a matrix strategy](https://github.com/orgs/community/discussions/37566)
> * [Running specific strategy on a different host](https://github.com/orgs/community/discussions/37815)
> * [How to validate that the user that issued a comment belongs to a specific github team](https://github.com/orgs/community/discussions/37769)
> * [Referencing the job calling a reusable workflow](https://github.com/orgs/community/discussions/37414)
> * [Resolving an issue where the environment shown in GitHub only shows the latest commit and not a list of commits](https://github.com/orgs/community/discussions/37391)
> * [How to Creare multi-variant matrix strategy from a step](https://github.com/orgs/community/discussions/37478)
> * [Step output playground - how to properly escape and write access to output variables](https://github.com/orgs/community/discussions/37477) ☮️
